guides/release/pages.yml

```
@@ -1,325 +1,325 @@
+    - title: "Spécifier le Modèle d'une Route"
       url: "specifying-a-routes-model"
-    - title: "Rendering a Template"
+    - title: "Rendre un Template"
       url: "rendering-a-template"
-    - title: "Redirecting"
+    - title: "Redirections"
       url: "redirection"
-    - title: "Preventing and Retrying Transitions"
+    - title: "Empêcher et Réessayer une Transition"
       url: "preventing-and-retrying-transitions"
       isAdvanced: true
-    - title: "Loading / Error Substates"
+    - title: "Chargement / Sous-États d'Erreur"
       url: "loading-and-error-substates"
       isAdvanced: true
-    - title: "Query Parameters"
+    - title: "Query Params"
       url: "query-params"
-    - title: "Asynchronous Routing"
+    - title: "Routing Asynchrone"
       url: "asynchronous-routing"
       isAdvanced: true
-    - title: "Controllers"
+    - title: "Contrôleurs"
       url: "controllers"
 - title: "Services"
   url: "services"
   pages:
-    - title: Overview
+    - title: Aperçu
       url: "index"
 - title: "Ember Data"
   url: "models"
   pages:
     - title: "Introduction"
       url: "index"
-    - title: "Defining Models"
+    - title: "Définir des Modèles"
       url: "defining-models"
-    - title: "Finding Records"
+    - title: "Requêter des Records"
       url: "finding-records"
-    - title: "Creating, Updating and Deleting"
+    - title: "Créer, Mettre à Jour et Supprimer"
       url: "creating-updating-and-deleting-records"
     - title: "Relationships"
       url: "relationships"
-    - title: "Pushing Records into the Store"
+    - title: "Pousser des Records dans le Store"
       url: "pushing-records-into-the-store"
       isAdvanced: true
-    - title: "Handling Metadata"
+    - title: "Gérer les Metadonnées"
       url: "handling-metadata"
       isAdvanced: true
-    - title: "Customizing Adapters"
+    - title: "Customiser les Adapters"
       url: "customizing-adapters"
       isAdvanced: true
-    - title: "Customizing Serializers"
+    - title: "Customiser les Serializers"
       url: "customizing-serializers"
       isAdvanced: true
-- title: "In-Depth Topics"
+- title: "Sujets Avancés"
   url: "in-depth-topics"
   pages:
-    - title: "Autotracking In-Depth"
+    - title: "L'Autotracking"
       url: "autotracking-in-depth"
-    - title: "Patterns for Components"
+    - title: "Patterns liés aux Composants"
       url: "patterns-for-components"
-    - title: "Patterns for Actions"
+    - title: "Patterns liés aux Actions"
       url: "patterns-for-actions"
-    - title: "Making API Requests"
+    - title: "Faire des Requêtes API"
       url: "making-api-requests"
-    - title: "Native Classes In-Depth"
+    - title: "Classes Natives"
       url: "native-classes-in-depth"
 
-- title: "Application Development"
+- title: "Développement d'une Application"
   url: "toc-heading_application-development"
   is_heading: true
-- title: "Application Concerns"
+- title: "À propos des Applications"
```

Not that the current change is wrong, just suggesting:

```suggestion
- title: "Applications - Notions importantes"
```